Sources/ComposableArchitecture/Reducer/AnyReducer/AnyReducerSignpost.swift

```diff
@@ -173,7 +176,7 @@ func debugCaseOutput(_ value: Any) -> String {
   }
 
   return (value as? CustomDebugStringConvertible)?.debugDescription
-    ?? "\(typeName(type(of: value)))\(debugCaseOutputHelp(value))"
+    ?? "\(abbreviated ? "" : typeName(type(of: value)))\(debugCaseOutputHelp(value))"
```

Comment:
Snuck another change in this PR to shorten the action description when you get a test failure telling you there are actions that were not yet received.